src/DynamoPackages/PackageManagerClient.cs

```diff
@@ -89,6 +89,20 @@ internal IEnumerable<PackageHeader> ListAll()
             }, new List<PackageHeader>());
         }
 
+        /// <summary>
+        /// Make a call to Package Manager to get the 
+        /// supported hosts for package publishing and filtering
+        /// </summary>
+        internal IEnumerable<string> GetSupportedHosts()
+        {
+            return FailFunc.TryExecute(() =>
+            {
+                var hosts = new Hosts();
+                var hostsResponse = this.client.ExecuteAndDeserializeWithContent<List<String>>(hosts);
```

Comment:
@pinzart Can you confirm this code from your example works? From my debugging the Greg Client call keeps giving me a failure thus a get an empty list

Reply:
@QilongTang Indeed there is an issue with the response format sent by PM server.
Fixing now...

Reply:
@QilongTang DEV should be fine now

Reply:
@pinzart Thanks, will continue my work today